demos/python_demos/object_detection_demo_yolov3_async/object_detection_demo_yolov3_async.py

```diff
@@ -310,8 +321,10 @@ def main():
         presenter.drawGraphs(frame)
         for obj in objects:
             # Validation bbox of detected object
-            if obj['xmax'] > origin_im_size[1] or obj['ymax'] > origin_im_size[0] or obj['xmin'] < 0 or obj['ymin'] < 0:
-                continue
+            obj['xmax'] = min(obj['xmax'], origin_im_size[1])
+            obj['ymax'] = min(obj['ymax'], origin_im_size[0])
+            obj['xmin'] = max(obj['xmin'], 0)
+            obj['ymin'] = max(obj['ymin'], 0)
```

Comment:
Shouldn't there still be some check as to whether the object overlaps the image at all?

Reply:
Box always intersects with image, because center of it **always** lies inside image.